dspace/config/modules/discovery.cfg

```diff
@@ -22,3 +22,7 @@ discovery.index.projection=dc.title,dc.contributor.*,dc.date.issued
 # 1) you need to set the DiscoverySearchRequestProcessor in the dspace.cfg 
 # 2) to show facet on Site/Community/etc. you need to add a Site/Community/Collection
 #	 Processors plugin in the dspace.cfg
+
+#Enable the url validation of the search.server setting above.
+#Defaults to true: validation is enabled
+#discovery.solr.url.validation.enabled = true
```

Comment:
@AndrewZWood : It looks like your last commit was an accident.  You seem to have replaced the setting with something unrelated :)